canonical/react-components
# fix: Getting started page error & update its GH links

## Done

- Fixes the Getting started page throwing an error (caused by missing import of `Meta` from `@storybook/addon-docs`)
- Updates the GH links at the bottom of the getting started page from `canonical-web-and-design` organization to `canonical`

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Go to the [getting started page for the demo](https://react-components-1092.demos.haus/?path=/docs/getting-started--docs)
- Verify that it does not throw an error and you can view the page contents, unlike [PROD](https://canonical.github.io/react-components/?path=/docs/getting-started--docs)
- Verify that the issues link on the getting started page points to the current Git repo issues page (https://github.com/canonical/react-components/issues)
- Verify that the contributing link on the getting started page points to the current Git repo page (https://github.com/canonical/react-components)

### Percy steps

- List any expected visual change in Percy, or write something like "No visual changes expected" if none is expected.

## Fixes

Fixes: #1064 

## Screenshots
Before:
![image](https://github.com/canonical/react-components/assets/46915153/6215be3f-368a-4622-be4f-7b770e65a34b)

After:
![image](https://github.com/canonical/react-components/assets/46915153/273a961e-af20-4329-bfdd-ef5f33ce44a3)
